### PR TITLE
Implement save state logic

### PR DESCRIPTION
**Pull request by ilukanets**
Please see https://github.com/h5p/h5p-documentation-tool/pull/8

### CODE REVIEW

**Comment by ilukanets** [comment]
This PR relates to https://github.com/h5p/h5p-documentation-tool/pull/10

**Comment by ilukanets** [comment]
This PR relates to https://github.com/h5p/h5p-documentation-tool/pull/10

File: standard-page.js [hunk 1]
```diff
@@ -144,6 +162,24 @@ H5P.StandardPage = (function ($, EventDispatcher) {
     });
   };
 
+  /**
+   * Sets previous state values for input fields
+   * @param state
+   */
+  StandardPage.prototype.setPreviousState = function (state) {
```
Comment (fnoks) [inline standard-page.js:169]: The previous state should be sent in through the constructor

File: standard-page.js [hunk 2]
```diff
@@ -144,6 +162,24 @@ H5P.StandardPage = (function ($, EventDispatcher) {
...
+    var inputIndex = 0;
+
+    this.pageInstances.forEach(function (instance) {
+      if (instance.libraryInfo.machineName === 'H5P.TextInputField' && instance.$inputField !== undefined) {
```
Comment (fnoks) [inline standard-page.js:173]: This feels wrong. A better solution would be to implement getCurrentstate in H5P.TextInputField

**Pull request closed without merging**